phpunit/data/themedir1/default/style.css

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Took the stylesheet from the `default` theme at [core](https://github.com/WordPress/wordpress-develop/tree/trunk/tests/phpunit/data/themedir1/default), so when this is ported tests don't need to be updated and work the same.